core/io/image_loader.cpp

```diff
@@ -62,7 +62,7 @@ Error ImageLoader::load_image(String p_file, Ref<Image> p_image, FileAccess *p_c
 
 		if (!loader[i]->recognize(extension))
 			continue;
-		Error err = loader[i]->load_image(p_image, f, p_force_linear);
+		Error err = loader[i]->load_image(p_image, f, p_force_linear, p_scale);
```

Comment:
@toger5 as i said also the force_linear does nothing in jpeg, for example, in this case scale seems worse, but keep in mind that the interaction point with these loaders is via the import dialog, so we just need to restrict the scale option to vector formats, so i propose adding a `is_vector` variable to image loaders so we know when scale can be applied, in that way we can hide the scale property when no needed.